php/scripts/docker-entrypoint.sh

```diff
@@ -28,7 +28,10 @@ if [ "$BEHAT_SELENIUM_HOST" != "" ] && [ "$BEHAT_WEB_HOST" != "" ]; then
     logger "Copying behat.yml.dist to behat.yml and updating selenium and web hosts"
     if [ -f behat.yml.dist ]; then
         cp -f behat.yml.dist behat.yml
+        sed -i "s@javascript_session: selenium@javascript_session: ${MINK_DEFAULT_SESSION}@" behat.yml
         sed -i "s@localhost:4444@${BEHAT_SELENIUM_HOST}:4444@" behat.yml
+        sed -i "s@localhost:4444@${BEHAT_SELENIUM_HOST}:4444@" behat.yml
+        sed -i "s@localhost:9222@${BEHAT_CHROMIUM_HOST}:9222@" behat.yml     
```

Comment:
Maybe have a default fallback on these new envs? (MINK_DEFAULT_SESSION and BEHAT_CHROMIUM_HOST)

Reply:
You're right - I was thinking that they will be defined in metarepo (added in corresponding PR to .env), but they don't have to be - I'll take that into account soon

Reply:
Addressed in https://github.com/ezsystems/docker-php/pull/39/commits/80bd5f09ac052191342ce5a529c8516295708b6d - Ive decided not to set any default values here, as they should be already present in behat.yml.dist